examples/subprotocol_server_youi/subprotocol_server_youi.cpp

```diff
@@ -0,0 +1,90 @@
+#include <iostream>
+#include <vector>
+#define ASIO_STANDALONE
+#include <websocketpp/config/asio_no_tls.hpp>
```

Comment:
Would put a space between the define
```suggestion
#include <vector>

#define ASIO_STANDALONE

#include <websocketpp/config/asio_no_tls.hpp>
```